### ISSUE TITLE
feature_request(enhancement): images detecting

### ISSUE THREAD

**Issue by Kristinita** [alvesjtiago/hover-preview#31]
### 1. Summary

It would be nice, if Hover Image Preview will work with links to images in any markup formats.

### 2. Argumentation

Hover Image Preview successful works for me in Markdown and HTML.

But for [**our project**](https://github.com/Kristinita/Erics-Green-Room) we use another format: image link end before any space symbol or asterisk — `*`. Regular expression for images in our project:

```regex
https?:\/\/[^\s\*]+\.(apng|bmp|gif|ico|jpeg|jpg|png|svg|svgz|webp)(?=(\s|\*))
```

Hover Image Preview doesn't work for these images.

### 3. Data

For example, I have a file `Kira__HoverImagePreview--Demo.md`. It content:

```markdown
![Kira Goddess](https://pp.userapi.com/c845421/v845421389/1541e3/xapzaLFRNFo.jpg)

https://pp.userapi.com/c845421/v845421389/1541e3/xapzaLFRNFo.jpg Kira Goddess
```

URLs is the same.

### 4. Steps to reproduce

I reproduce the problem in a version of Sublime Text without plugins and user settings.

### 5. Expected behavior

If I hover cursor to first link:

![Expected](https://i.imgur.com/5e049vI.png)

Success. Image are displayed.

### 6. Actual behavior

![Actual](https://i.imgur.com/gWCK2E8.png)

No effect.

### 7. Environment

+ Windows 10 Enterprise LTSB 64-bit EN
+ Sublime Text Build 3176
+ Hover Image Preview 0.0.21

Thanks.

**Comment by AmjadHD** [comment]
currently the plugin only works for urls enclosed in `()`, `""` and `''`, a good regex that matches only image urls in arbitrary formats is little tricky to figure, any help will be appreciated.

**Comment by theBowja** [comment]
perhaps implement highlight+hover as a workaround?

**Comment by AmjadHD** [comment]
> perhaps implement highlight+hover as a workaround?
#36 implements highlight+hover

**Comment by AmjadHD** [comment]
> It would be nice, if Hover Image Preview will work with links to images in any markup formats.

#37

**Comment by alvesjtiago** [comment]
This is addressed with #37 and version 0.1.0 which should be automatically updated via package manager.
